stdlib/testing/testing.flux

```diff
@@ -9,27 +9,20 @@ builtin diff
 option loadStorage = (csv) => c.from(csv: csv)
 option loadMem = (csv) => c.from(csv: csv)
 
-run = (case) => {
-    tc = case()
-    return tc.input
-        |> tc.fn()
-        |> yield(name: "_test_result")
-        |> diff(want: tc.want)
-        |> yield(name: "diff")
-        |> assertEmpty()
```

Comment:
So previously assertEmpty could fail before all tables were processed in which case the diff will be incomplete?